values.yaml

```diff
@@ -111,6 +111,33 @@ global:
     # clients and servers and only accept HTTPS connections.
     httpsOnly: true
 
+    # caCert is a kubernetes secret containing the certificate
+    # of the CA to use for TLS communications within the Consul cluster.
+    # If you have generated the CA yourself with the consul CLI,
+    # you could use the following command to create the secret in Kubernetes:
+    #
+    #   kubectl create secret generic consul-ca-cert \
+    #           --from-file='tls.crt=./consul-agent-ca.pem'
+    caCert:
+      secretName: ""
```

Comment:
We've been using `null` for these in other places

Reply:
I think I looked at gossip encryption which happens to be the only one that uses empty strings 😄 . It seems like `null` is more common; I'll update.